index.ts

```diff
@@ -661,23 +661,30 @@ export class EthereumBlock {
   timestamp: BigInt
   difficulty: BigInt
   totalDifficulty: BigInt
+  size: BigInt | null
 }
 
 /**
  * An Ethereum transaction.
  */
 export class EthereumTransaction {
   hash: Bytes
-  blockHash: Bytes
-  blockNumber: BigInt
+  index: BigInt
+  from: Address
+  to: Address | null
+  value: BigInt
   gasUsed: BigInt
+  gasPrice: BigInt
 }
 
 /**
  * Common representation for Ethereum smart contract events.
  */
 export class EthereumEvent {
   address: Address
+  logIndex: BigInt
+  transactionLogIndex: BigInt
+  logType: String | null
```

Comment:
We're using `string` everywhere else and I believe there are some subtle differences between `String` and `string` in AssemblyScript. So let's change this to `string`.